e2e/utils/components/promotion-carousel.component.ts

```diff
@@ -0,0 +1,36 @@
+import { CAROUSEL_CONTAINER, CAROUSEL_SLIDE_NAVIGATION } from '$lib/constants/test-ids.constants';
+import { Locator, Page } from '@playwright/test';
+
+export class PromotionCarousel {
+	#page: Page;
+	#container: Locator;
+	#slideNavigation: Locator;
+
+	constructor(page: Page) {
+		this.#page = page;
+		this.#container = this.#page.locator(`[data-tid="${CAROUSEL_CONTAINER}"]:visible`);
+		this.#slideNavigation = this.#page.locator(`[data-tid="${CAROUSEL_SLIDE_NAVIGATION}"]:visible`);
+	}
+
+	public async navigateToSlide(slideNumber: number): Promise<void> {
+		const navigation1Selector = `[data-tid="carousel-slide-navigation-${slideNumber}"]`;
+		await this.#page.click(navigation1Selector);
+	}
+	public async freezeCarousel(): Promise<void> {
+		await this.#page.$$eval(`div[data-tid="carousel-slide"]`, (elements) => {
+			elements.forEach((el) => {
+				const slideStyle = el.style.transform;
```

Comment:
correct, because we don't want to touch setInterval for the whole application we can override it like this. Basically after 5 Seconds the autoplay wants to change to the next slide but we just always put the values of the first slide back in.